docs/decisions/0011-depending-on-multiple-event-bus-implementations.rst

```diff
@@ -0,0 +1,86 @@
+11. Depending on multiple event bus implementations
+###################################################
+
+Status
+******
+
+**Accepted** *2023-05-25*
+
+(This decision was actually made in November 2022, and retroactively documented in May 2023 but with additional, newer context such as the existence of event-bus-redis.)
+
+Context
+*******
+
```

Comment:
Consider adding a link to the OEP for additional context of Kafka and Redis, since the ADRs are available from the OEP. https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0052-arch-event-bus-architecture.html